### PR TITLE
Start box hunt at <= minimum box setting instead of < box_limit

### PR DESCRIPTION
**Pull request by jbrock24**
Added in a start point, so you hunt up to a certain amount of boxes, but don't go on the hunt unless you're near empty, empty, player defined start, etc.

### CODE REVIEW

File: hunting-buddy.lic [hunk 1]
```diff
@@ -43,7 +43,7 @@ class HuntingBuddy
 
   def need_boxes?
     return false unless @settings.box_loot_limit
```
Comment (rpherbig) [inline hunting-buddy.lic]: Should this check for `box_hunt_minimum`?

File: profiles/Ssarek-setup.yaml [hunk 2]
```diff
@@ -152,441 +169,435 @@ buff_spells: &buff_spells
...
-  Utility:
-    abbrev: hw
-    symbiosis: true
-    cast: cast chest
-    mana: 15
+###############
+# Locksmithing
+###############
+box_loot_limit: 5
+box_hunt_minimum: 1 # Private script
```
[the remark this answers is not in the document]
Comment (jayRyan24) [reply profiles/Ssarek-setup.yaml]: Made the change.

File: hunting-buddy.lic (hunk 1 above)
Comment (jayRyan24) [reply hunting-buddy.lic]: Made the change.

**Comment by jayRyan24** [comment]
@rpherbig

**Pull request merged**